dotnet-base-libraries/CryptographyRandomNumberGenerator/App/RandomNumberBenchmarks.cs

```diff
@@ -0,0 +1,21 @@
+using BenchmarkDotNet.Attributes;
+
+namespace App;
+
+public class RandomNumberBenchmarks
+{
+    [Benchmark]
+    public void UsingRealRandomNumberGenerator()
+    {
+        var randomNumber = CryptographicHelpers.GenerateRandomInteger(1, 100);
```

Comment:
We should add the return type and return this value as it will prevent the JIT from changing this to a noop. It is not so much a problem anymore, but is still considered a best practice with benchmarks to ensure the method is not nooped.